pkg/boot/params.go

```diff
@@ -118,13 +122,26 @@ type Params struct {
 	LocalPK          cipher.PubKey `json:"local_pk"` // Not actually encoded to bps.
 	LocalSK          cipher.SecKey `json:"local_sk"`
 
+	// only on dmsghttp
+	DMSGHTTP string `json:"dmsghttp,omitempty"`
+
 	// only valid if mode == "0x00" (hypervisor)
 	HypervisorPKs    cipher.PubKeys `json:"hypervisor_pks,omitempty"`
 	SkysocksPasscode string         `json:"skysocks_passcode,omitempty"`
 }
 
+type dmsgHTTPServers struct {
```

Comment:
Maybe we can move it to pkg, export and use [`dmsgHTTPServersData`](https://github.com/ersonp/skywire/blob/d18280346f59a769fd008057da76983896b5daaf/cmd/skywire-cli/commands/config/gen.go#L195) from `skywire` here.

Reply:
https://github.com/skycoin/skywire/pull/1009